desc/compute/_core.py

```diff
@@ -28,7 +46,7 @@ def _0(params, transforms, profiles, data, **kwargs):
     label="\\rho",
     units="~",
     units_long="None",
-    description="Radial coordinate, proportional to the square root "
+    description="Radial coordinate, proportional to the square root"
```

Comment:
this space is actually necessary, since the sentence continues on the next line.